quickwit-metastore/src/metastore/mod.rs

```diff
@@ -172,9 +172,9 @@ pub trait Metastore: Send + Sync + 'static {
 
 /// Finds out if a split matches a tag filter expression.
 ///
-/// `filter_tags` tags is an array representing a boolean expression
+/// `filter_tags` is an array representing a boolean expression
```

Comment:
I think we can describe a little bit better, `filter_tags` is an array of array of tags representing
a boolean expression 
`of the form [ [A OR B OR...] AND [C OR D OR ...] AND ...]